subprojects/docs/src/docs/release/notes.md

```diff
@@ -162,6 +163,16 @@ BUILD SUCCESSFUL in 550ms
 1 actionable task: 1 executed 
 ```
 
+### Run a single PMD task on multiple threads
+
+[PMD](https://pmd.github.io/) is a quality analysis tool that runs on the Java source files of your project.
+
+With this version of Gradle, the [`thread` parameter](https://pmd.github.io/latest/pmd_userdocs_tools_ant.html#parameters) it offers is now exposed through the PMD extension and tasks.
+This allows configuration of PMD to run its analysis on more than one thread.
+Note that this setting is independent of the [number of workers](userguide/performance.html#parallel_execution) used by Gradle.
```

Comment:
It would be good to mention the practical implications of this fact either here or in the documentation.

Reply:
Can you be more specific? I am not sure what you are asking for.

Reply:
I'm referring to the fact that the thread parameter is independent of the number of workers. What does it mean for the user? If PMD tasks can already run in parallel without the thread parameter, should users be mindful of this and not set the thread parameter too high, risking too much parallelism and degraded performance?

Reply:
I moved that information to the PMD chapter in docs. PTAL